### DISCUSSION TITLE
Created a Redesigned version of the portfolio-website

### DISCUSSION THREAD

**Discussion by itsarghyadas** [EddieHubCommunity/support#5576] [Show and tell]
@kaiwalyakoparkar I saw your portfolio website and I thought it is looking a bit outdated, so I made a redesigned version of it
Here is the link - https://www.behance.net/gallery/153897829/Kaiwalya-Koparkar-Portfolio
Although it is not perfect, we can work together to make something cool and implement it on your website.

**Comment by kaiwalyakoparkar** [comment]
@itsarghyadas loved it, can I use it to rewamp my current website design. I have been thinking that for a while now?

**Reply by itsarghyadas** [reply]
Yes yes, you can use it, that's why I made it. See once more, I updated it, I don't know if you saw the updated one or the old one.😅
We can make it better if we can connect and work with what design you have in your mind, If you want I can try to create the whole website version, just don't use this font, I guess, use some simple fast fonts like 'Inter', so your website can be loaded-fast. And thanks for the feedback @kaiwalyakoparkar

**Reply by kaiwalyakoparkar** [reply]
@itsarghyadas I would be really grateful if you can create entire website version for me and tell me the info like colors, fonts you used (If that's okay)

**Reply by itsarghyadas** [reply]
@kaiwalyakoparkar It would be great if we could connect and directly chat on any social platform like Twitter or LinkedIn wherever you like.

**Reply by itsarghyadas** [reply]
Yes yes, no problem, I will share all the info about the elements I used on the website after I create the entire website, you can save that Behance link, I will be updating it there.
